libs/chain-signatures/contract/src/lib.rs

```diff
@@ -222,6 +257,7 @@ impl VersionedMpcContract {
         const CHEAP_REQUESTS: u32 = 3;
         let pending_requests = match self {
             Self::V0(mpc_contract) => mpc_contract.request_counter,
+            Self::V1(_) => 0,
```

Comment:
This is a bit convoluted, maybe just `return U128::from(1)` immediately here instead of faking `pending_requests` to get the behavior we want out of the cost function below.